lib/ui/view/channel/channel_info/channel_info_view_model.dart

```diff
@@ -7,10 +9,26 @@ import '../../../../app/app.router.dart';
 class ChannelInfoViewModel extends BaseViewModel {
   final _navigationService = locator<NavigationService>();
 
+  final _dialogService = locator<DialogService>();
+
+
+
+  Future showDialog() async{
+    print("Testing call");
```

Comment:
remove print statement, use logger if you want to print vars

Reply:
resolved.